.github/workflows/cron-flakes.yml

```diff
@@ -58,11 +58,7 @@ jobs:
 
     - name: Import ${{ matrix.group }} group
       run: |
-        ./result/bin/flake-info --push --json group ./flakes/${{ matrix.group }}.json ${{ matrix.group }} 1>${{ matrix.group }}.json
-
-        # todo cache ES json?
-
-
+        ./result/bin/flake-info --push --elastic-schema-version=$(cat ./VERSION) group ./flakes/${{ matrix.group }}.json
```

Comment:
What is wrong about reading the version from the Environment?

Reply:
You can not cat from file before checkout job :)